events/app/views/pages/events.njk

```diff
@@ -0,0 +1,34 @@
+{% extends 'layout/default.njk' %}
+
+{% block pageMeta %}
+  {% set metaContent = {
+    type: 'website',
+    title: 'Events',
+    image: allEvents[0].image.contentUrl,
+    url: pageConfig.canonicalUri,
+    description: 'Past and present events at Wellcome Collection'
+  } %}
+  {% component 'open-graph', { pageConfig: pageConfig, metaContent: metaContent } %}
+  {% component 'twitter-card', { pageConfig: pageConfig, metaContent: metaContent } %}
+
+ <meta name="description" content="{{ metaContent.description }}" />
+{% endblock %}
+
+{% block body %}
+
+  <script type="application/ld+json">
+    {# TODO: add JSON+LD #}
```

Comment:
Could this be part of the PR given that SEO is P1?
Or definitely the next step?

Reply:
https://github.com/wellcometrust/wellcomecollection.org/issues/1892

Reply:
@jamesgorrie I understood that you were tackling the Events JSON-LD stuff, so didn't want to duplicate the work. I think I got that from Slack – perhaps I misinterpreted something. I'll do it if you're not?